client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -587,6 +636,24 @@ export const useCommandsArrayWpcom = ( {
 			},
 			icon: pluginsIcon,
 		},
+		{
+			name: 'installPlugin',
+			label: __( 'Install plugin' ),
```

Comment:
The question about the "Install theme" applies here too.

In my scenario, I test both the "Install plugin" and "Manage plugins" actions and they point to the same page.

Reply:
If we are going to keep both, should we add some keywords, e.g., "add a plugin" and maybe "plugins"?